concent_api/common/helpers.py

```diff
@@ -204,6 +205,12 @@ def generate_ethereum_address_from_ethereum_public_key(ethereum_public_key: str)
     return generate_ethereum_address_from_ethereum_public_key_bytes(ethereum_public_key).hex()
 
 
+def ethereum_public_key_to_address(ethereum_public_key: str) -> str:
+    return to_checksum_address(
+        sha3(decode_hex(ethereum_public_key))[12:].hex()
+    )
```

Comment:
Please provide at least a rudimentary unit test for this function on a known pair of a key and address.

Reply:
:+1: done